examples/cmake_example/main.cpp

```diff
@@ -1,13 +1,17 @@
 #include "arkCrypto.h"
 
+#include <cstdint>
 #include <iostream>
 #include <string>
 #include <vector>
-#include <cstdint>
 
 int main(int argc, char* argv[]) {
-  const auto text = "Computer science is no more about computers than astronomy is about telescopes.";
-  const auto passphrase = "viable weasel wage promote praise inflict jaguar tackle color unusual exclude direct";
+  const auto text =
+      "Computer science is no more about computers than astronomy is about "
+      "telescopes.";
+  const auto passphrase =
+      "viable weasel wage promote praise inflict jaguar tackle color unusual "
+      "exclude direct";
```

Comment:
The google profile for clang format has a max line length (i would assume 80 characters).

All this is can be changed but I figured we would get our codebase up to a standard format and then we could have a discussion if we like it.

Reply:
It seems to be how it handles line-length though.
I wouldn't think splitting a string like that should even compile, and it doesn't make sense.

Other issues involve comments going past the line limit, and split up an otherwise good line.